src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const bodyDoc = document.querySelector('body');
+const mateLogo = document.querySelector('.logo');
+
+const promise1 = () => {
+  return new Promise((resolve, reject) => {
+    mateLogo.addEventListener('click', () => resolve('Promise was resolved!'));
+  });
+};
+
+const promise2 = () => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
```

Comment:
The `setTimeout` function in `promise2` will always reject the promise after 3 seconds, regardless of any other interactions. If this is the intended behavior, it is fine; otherwise, you may want to add a condition for the rejection.